doc/how-to/how-to-add-allow-list-variables.md

```diff
@@ -0,0 +1,111 @@
+
+  Example to look for:
+
+```sql
+  /* Example from submission_services_summary */
+  (((ls.allow_list_answers -> 'application.type'::text) -> 0))::text AS application_type
+
+  /*Example from analytics_summary */
+  (((al.allow_list_answers -> 'application.type'::text) -> 0))::text AS application_type
+```
+
+  Example of how a view may look:
+
+```sql
+  CREATE OR REPLACE VIEW "public"."submission_services_summary" AS 
+  WITH resumes_per_session AS
+  /* deleted the code here for readability */
+  SELECT (ls.id)::text AS session_id,
+      t.slug AS team_slug,
+      f.slug AS service_slug,
+      ls.created_at,
+      ls.submitted_at,
+      ((ls.submitted_at)::date - (ls.created_at)::date) AS session_length_days,
+      ls.has_user_saved AS user_clicked_save,
+      rps.number_times_resumed,
+      ls.allow_list_answers,
+      ((ls.allow_list_answers -> 'proposal.projectType'::text))::text AS proposal_project_type,
+      ((ls.allow_list_answers -> 'application.declaration.connection'::text))::text AS application_declaration_connection,
+      ((ls.allow_list_answers -> 'property.type'::text))::text AS property_type,
+      ((ls.allow_list_answers -> 'drawBoundary.action'::text))::text AS draw_boundary_action,
+      ((ls.allow_list_answers -> 'user.role'::text))::text AS user_role,
+      ((ls.allow_list_answers -> 'property.constraints.planning'::text))::text AS property_constraints_planning,
+      /* deleted the code here for readability */
+      sa.s3_applications,
+      ((ls.allow_list_answers -> 'usedFOIYNPP'::text))::text AS used_foiynpp,
+      ((ls.allow_list_answers -> 'propertyInformation.action'::text))::text AS property_information_action,
+      ((ls.allow_list_answers -> 'planningConstraints.action'::text))::text AS planning_constraints_action,
+      ((ls.allow_list_answers -> '_overrides'::text))::text AS overrides,
+      ((ls.allow_list_answers -> 'rab.exitReason'::text))::text AS rab_exit_reason,
+      ((ls.allow_list_answers -> 'service.type'::text))::text AS pre_app_service_type,
+      ((ls.allow_list_answers -> 'application.information.harmful'::text))::text AS pre_app_harmful_info,
+      ((ls.allow_list_answers -> 'application.information.sensitive'::text))::text AS pre_app_sensitive_info,
+      (((ls.allow_list_answers -> 'application.type'::text) -> 0))::text AS application_type
+    FROM lowcal_sessions ls
+    /* deleted the code here for readability */
+    WHERE ((f.slug IS NOT NULL) AND (t.slug IS NOT NULL));
+```
+
+  The values here are being pulled from the table `lowcal_sessions.allow_list_answers` or `analytics_logs.allow_list_answers`
+
+  **At the end of your SQL script after the view creation/replacement, it is important to add another line after each one.**
+
+  `GRANT SELECT ON public.{VIEW_NAME} TO metabase_read_only`
+
+  So, your migration may end with something like this:
```

Comment:
Maybe make this a bit more definite, like `We currently have two views, so you should add these two lines to the end of the migration file:`